modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/idprovider/IdProviderResponseWrapper.java

```diff
@@ -124,43 +129,30 @@ public void sendError( final int sc, final String msg )
     }
 
     private void handleError( final int sc )
+        throws IOException
```

Comment:
Why should we handle this in an external method if we could have thrown the  `UncheckedIOException`  error right here?

Reply:
most usages already can handle IOException, only one that doesn't - handled with UncheckedIOException